### ISSUE TITLE
Public Tracker support (extratorrent.cc, eztv.ag)

### ISSUE THREAD

**Issue by seertenedos** [cardigann/cardigann#9]
Any chance of adding public tracker support at well? While i know that most apps take in RSS these sites also let you search and if they can be wrapped in torznab then most of the apps would auto support them. There is not currently any torznab app that i know of that supports the public trackers and i find the support quite spotty in the apps like sonnar that support public trackers.  I am still trying to get into the private ones :(

**Comment by ldexterldesign** [comment]
Hey,

http://opentrackers.net/
http://opentrackers.org/
http://reddit.com/r/opensignups/
http://torrent-invites.com/
http://torrentinvitez.com/

Hope this is useful

Regards

**Comment by lxt2** [comment]
@seertenedos yup, if you wanted to provide a list of public trackers, I'd consider supporting them.

**Comment by seertenedos** [comment]
extratorrent.cc, eztv.ag, and http://www.nyaa.se
plus the ones @ldexterldesign mentioned

**Comment by lxt2** [comment]
I've added TPB and intend to add the others.

**Comment by lxt2** [comment]
eztv.ag is added.

**Comment by zductiv** [comment]
Still experiencing issues with eztv.ag on version 1.8.3

When adding, the state is returned as OK. However, if you click on test it will return:

"An error occurred.
Expected no results, got 5"

and if you add the feed to Sonarr you will get a 0/0 seed/peer count for any torrents from eztv.

**Comment by CyberKor** [comment]
Awesome work Ixt2!!
Yes, ExtraTorrent.cc add would be super, as well as Zooqle :)

**Comment by BetaLeaf** [comment]
Great works @lxt2. Thanks.

**Comment by ldexterldesign** [comment]
See https://github.com/cardigann/cardigann/issues/324#issuecomment-331602043
